src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/MathFunctionsIT.java

```diff
@@ -138,6 +138,42 @@ public void sinh() throws IOException {
         assertThat(sinh, equalTo(Math.sinh(Math.PI)));
     }
 
+    @Test
+    public void power() throws IOException {
+        SearchHit[] hits = query(
+                "SELECT POWER(PI(), 2) AS power"
+        );
+        double power = (double) getField(hits[0], "power");
+        assertThat(power, equalTo(Math.pow(Math.PI, 2)));
+    }
+
+    @Test
+    public void atan2() throws IOException {
+        SearchHit[] hits = query(
+                "SELECT ATAN2(age, age) AS atan2", "WHERE age IS NOT NULL"
+        );
+        double atan2 = (double) getField(hits[0], "atan2");
+        assertThat(atan2, equalTo(Math.atan2(1, 1)));
+    }
+
+    @Test
+    public void cot() throws IOException {
+        SearchHit[] hits = query(
+                "SELECT COT(PI()) AS cot"
+        );
+        double cot = (double) getField(hits[0], "cot");
+        assertThat(cot, equalTo(1 / Math.tan(Math.PI)));
```

Comment:
Matcher `closeTo` is safer for double. I recall there were some test case that failed occasionally due to precision issue.

Reply:
@dai-chen Good, that sounds more proper to use `closeTo`. will do it, thanks!